MANIFEST.in

```diff
@@ -4,3 +4,4 @@
 
 exclude .gitignore
 exclude .gitlab-ci.yaml
+exclude .vscode
```

Comment:
`.vscode` should not be under version control (and thus should be specified in `.gitignore`, which I think it is), which means that it doesn't need to be explicitly specified here as long as `setuptools-git` is used in `setup.py` (which I think it is). Check the comment in lines 1-3 for more details.